### PR TITLE
feat: Added switching between groups

### PR DESCRIPTION
**Pull request by jotjern**

### CODE REVIEW

File: frontend/src/views/groups/GroupsView.tsx [hunk 1]
```diff
@@ -22,20 +21,31 @@ export const GroupsView = () => {
       }),
   });
 
+  useEffect(() => {
+    if (user && props.selectedGroupId === undefined) {
+      console.log("hei");
```
Comment (Terbau) [inline frontend/src/views/groups/GroupsView.tsx:26]: Hei

**Review by Terbau — CHANGES_REQUESTED** [review]
Overall very nice!!

File: frontend/src/views/groups/GroupsView.tsx (hunk 1 above)
Comment (jotjern) [reply frontend/src/views/groups/GroupsView.tsx]: hei

**Review by Terbau — APPROVED** [review]
LGTM

**Pull request merged**